packages/wrangler/src/pages/dev.tsx

```diff
@@ -87,6 +87,11 @@ export function Options(yargs: Argv) {
 				description:
 					"The location of the single Worker script if not using functions",
 			},
+			functions: {
```

Comment:
Any reason you pulled this out? It's not currently customizable on prod. I'd _slightly_ prefer waiting until we had it there before adding here, but I'm not hugely fussed. If you have a need for it now, absolutely fine.

Reply:
Must be a merge snafu? I didn't intend to touch this at all...